muxers/yamux/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.45.1
```

Comment:
@mxinden You forgot to bump the version in `Cargo.toml` or this. CI didn't catch it because this check is disabled for dependency upgrades as we don't want to bump the version for dependencies by default.